furusiya/model/components/fighter.py

```diff
@@ -62,4 +52,23 @@ def heal(self, amount):
         # heal by the given amount, without going over the maximum
         self.hp += amount
         if self.hp > self.max_hp:
-            self.hp = self.max_hp
\ No newline at end of file
+            self.hp = self.max_hp
+
+    def die(self):
+        # Drop arrows if it's a monster
+        if config.data.features.limitedArrows and AISystem.has_ai(self.owner):
```

Comment:
I just realized that your horse will drop arrows when it dies. I wonder if this is a bug or a strategic feature?!